docs/docs/features/studio.md

```diff
@@ -2,14 +2,20 @@
 title: ZMK Studio
 ---
 
-:::warning[Alpha Feature]
+:::warning[Beta Feature]
 
-ZMK Studio support is in alpha. Although best efforts are being made, keeping compatibility during active development is not guaranteed.
+ZMK Studio support is in beta. Although every effort has been made to provide a stable experience, you may still encounter issues during use. Please report any issues to [GitHub Issues](https://github.com/zmkfirmware/zmk-studio/issues)
 
 :::
 
 ZMK Studio provides runtime update functionality to ZMK powered devices, allowing users to change their keymap layers without flashing new firmware to their keyboards. Studio is still under active development, and is not yet ready for casual end user use.
 
+## Accessing ZMK Studio
+
+If using Chrome/Edge, you can use ZMK Studio directly from your browser at https://zmk.studio/
+
+To use the native app for Linux, macOS, or Windows, download the appropriate file from the [latest release](https://github.com/zmkfirmware/zmk-studio/releases)
```

Comment:
```suggestion
If using Chrome/Edge, you can use ZMK Studio directly from your browser at [https://zmk.studio/](https://zmk.studio/).

To use the native app for Linux, macOS, or Windows, download the appropriate file from the [latest release](https://github.com/zmkfirmware/zmk-studio/releases).
```
Weird link in a link because I'm not sure how it'll handle the period.

Reply:
Added it I think as you were commenting. I didn't do the double link, it does handle the `.` properly.